pkg/appconsts/global_consts.go

```diff
@@ -30,7 +30,10 @@ const (
 	// DefaultUpgradeHeightDelay is the number of blocks after a quorum has been
 	// reached that the chain should upgrade to the new version. Assuming a block
 	// interval of 12 seconds, this is 7 days.
-	DefaultUpgradeHeightDelay = int64(7 * 24 * 60 * 60 / 12) // 7 days * 24 hours * 60 minutes * 60 seconds / 12 seconds per block = 50,400 blocks.
+	// DefaultUpgradeHeightDelay = int64(7 * 24 * 60 * 60 / 12)  // 7 days * 24 hours * 60 minutes * 60 seconds / 12 seconds per block = 50,400 blocks.
+
+	// HACKHACK: Override the default for the e2e test. Do not merge this as-is.
+	DefaultUpgradeHeightDelay = int64(1)
```

Comment:
I think the only remaining thing on this PR is to revert this change and plumb an override for the upgrade height via build flags (or something else).

cc: @cmwaters

Reply:
we can fix this after the PR merges in https://github.com/celestiaorg/celestia-app/issues/3947